.github/workflows/maven.yml

```
@@ -0,0 +1,33 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

also wouldn't mind running all these Github actions on all branches:
```suggestion
on: [push, pull_request
```